### DISCUSSION TITLE
Lost Key Email

### DISCUSSION THREAD

**Discussion by reubadoob** [aclap-dev/video-downloadhelper#458] [Help and Q&A]
### Your question

What's your issue?

I lost the email with my license key. Could you resend it, please? My registered email is reubadoob@gmail.com 

Thank you

The 10 first digits (only 10) of your license number or email address
reubadoob@gmail.com 

### Addon & CoApp version and details

version: 8.2.2.8
channel: stable
build date: 2024-03-18
build options: linuxlic: false, noyt: false, browser: firefox
lang: en-US
license: unset
platform: x86-64 win
UA: Mozilla/5.0 (Windows NT 10.0; Win64; x64; rv:125.0) Gecko/20100101 Firefox/125.0
coapp: yes. Info:
{
  "id": "net.downloadhelper.coapp",
  "name": "VdhCoApp",
  "version": "2.0.19",
  "binary": "C:\\Program Files\\DownloadHelper CoApp\\vdhcoapp.exe",
  "displayName": "VdhCoApp",
  "description": "Video DownloadHelper companion app",
  "target": {
    "os": "windows",
    "arch": "x86_64",
    "node": "18"
  },
  "home": "C:\\Users\\reuba",
  "converterBinary": "C:\\Program Files\\DownloadHelper CoApp\\ffmpeg.exe",
  "converterBase": "ffmpeg",
  "converterBaseVersion": "c30f360955"
}

**Comment by jcfrog** [comment]
HI,
We sent you an email
